src/bundle/Resources/public/js/scripts/button.state.checkbox.toggle.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc) {
+    const toggleForms = doc.querySelectorAll('.ez-toggle-btn-state-checkbox');
+    const ALL_CHECKED = 'all-checked';
+    const ANY_CHECKED = 'any-checked';
+
+    const toggleButtonState = (toggleForm, checkboxInputs) => {
+        const button = doc.querySelector(toggleForm.dataset.toggleButtonId);
+        const anyIsChecked = checkboxInputs.some(el => el.checked);
+        const allChecked = checkboxInputs.every(el => el.checked);
```

Comment:
I think this line and the line above can be moved into the line line no. 22 and 23, for better performance.